sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -117,7 +117,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
     </module>
 
     <!-- Check that paragraph tags are used correctly in Javadoc. -->
-    <module name="JavadocParagraph"/>
+    <!-- <module name="JavadocParagraph"/> -->
```

Comment:
Can you add context why this needs to be disabled? And maybe a tracking JIRA if it should eventually be re-enabled?

Reply:
Done